### PR TITLE
Change target SDK to Android P, also upgrade Gradle and dependencies

### PR DESCRIPTION
**Pull request by babedev**
Upgrade target SDK to Android P (28) and Gradle 5.3.

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -3,8 +3,11 @@
   <component name="ProjectModuleManager">
     <modules>
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
+      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
```
Comment (nuxzero) [inline .idea/modules.xml]: Could you delete duplicated config?

Comment (Namiii) [reply .idea/modules.xml]: Think there is more than one duplicated config

File: sdk/build.gradle [hunk 2]
```diff
@@ -83,15 +83,15 @@ uploadArchives {
 }
 
 dependencies {
-    compile 'joda-time:joda-time:2.9.2'
-    compile 'com.squareup.okhttp3:okhttp:3.4.1'
-    androidTestCompile 'com.android.support:support-annotations:26.1.0'
-    androidTestCompile('com.android.support.test.espresso:espresso-core:3.0.1', {
+    api 'joda-time:joda-time:2.9.2'
```
Comment (Namiii) [inline sdk/build.gradle]: Any particular reason `api` is preferred to `implementation` here?

**Comment by nuxzero** [comment]
Closed, this PR duplicated with #55.

**Pull request closed without merging**